homeworks/src/ltretykhlib/module3/Triangle.java

```diff
@@ -0,0 +1,98 @@
+package ltretykhlib.module3;
+
+import static java.lang.Math.*;
+
+/**
+ * Created by Tretykhlib on 23.04.2017.
+ */
+public class Triangle {
+    private double Ax;
+    private double Ay;
+    private double Bx;
+    private double By;
+    private double Cx;
+    private double Cy;
+
+    public Triangle(double axTop, double ayTop, double bxTop, double byTop, double cxTop, double cyTop) {
+        Ax = axTop;
+        Ay = ayTop;
+        Bx = bxTop;
+        By = byTop;
+        Cx = cxTop;
+        Cy = cyTop;
+    }
```

Comment:
Make to the constructor to take 3 Points instead.
Add validation ( create a separate method to check if a given triangle can exist) to the constructor.